contrib/sync-upstream.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+set -eou pipefail
+
+help() {
+    echo "$0 range <branch> <startcommit> [end]"
+    echo "    merges every merge commit in the range from the <start> merge commit (excluding) to the optional [end] commit (including)."
+    echo
+    echo "$0 select <commit> ... <commit>"
+    echo "    merges every selected merge commit"
+    echo
+    echo "Note that this tool requires the github-cli tool (aka gh)."
```

Comment:
Maybe say that this is only to create the final PR (and the user could do this manually).

Also it would be good to explain that invoking this script will only create a branch with name foo and not already create a PR. (To make sure people know they can play around with it locally.)

Reply:
Fixed